Wiley Online Library.js

```diff
@@ -229,7 +229,14 @@ function scrapeBibTeX(doc, url, pdfUrl) {
 				'&fileFormat=REFERENCE_MANAGER' +
 				'&hasAbstract=CITATION_AND_ABSTRACT';
 	ZU.doPost(postUrl, body, function(text) {
-		//Z.debug(text)
+		//on at least one occasion, Wiley puts RIS within RIS, thus causing havoc.
+		//This is trying to prevent this (see last test).
+		textClean = text.split(/Related Articles\r/);
```

Comment:
Are you sure the regex works for ```Related Articles: ```? I see there also a colon and space and sometimes `\n`...